cordapp-cpk2/src/test/resources/cpk-export-reserved-package/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id 'net.corda.plugins.cordapp-cpk2'
+}
+
+apply from: 'repositories.gradle'
+apply from: 'javaTarget.gradle'
+
+group = 'com.example'
+version = cordapp_version
+
+cordapp {
+    targetPlatformVersion = platform_version.toInteger()
+    minimumPlatformVersion = platform_version.toInteger()
+
+    contract {
+        name = 'cpk-export-reserved-package'
+        versionId = cordapp_contract_version.toInteger()
+        licence = 'Test-Licence'
+        vendor = 'R3'
+    }
+}
+
+dependencies {
+    cordaProvided project(':corda-api')
+    implementation "commons-io:commons-io:$commons_io_version"
+}
+
+jar {
+    archiveBaseName = 'cpk-export-reserved-package'
```

Comment:
But suppose the CPK writer does this instead?
```gradle
jar {
    archiveBaseName = 'cpk-export-reserved-package'
    osgi {
        autoExport = false
    }
}
```
And provides a `package-info.java` too, of course:
```java
@Export
package net.corda.contract;

import org.osgi.annotation.bundle.Export;
```
And not forgetting to include this in the test:
```kotlin
.withSubResource("src/main/java/net/corda/contract/package-info.java")
```

Reply:
I've added this as a test and applied the fix suggested in the second comment.